src/ApparentHorizons/Tags.hpp

```diff
@@ -369,7 +369,10 @@ struct IrreducibleMass : db::SimpleTag {
 template <typename Frame>
 struct IrreducibleMassCompute : IrreducibleMass, db::ComputeTag {
   using base = IrreducibleMass;
-  static constexpr auto function = ::StrahlkorperGr::irreducible_mass;
+  static void function(const gsl::not_null<double *> result,
```

Comment:
No space between `double` and `*`

Reply:
All checks passed @geoffrey4444 @nilsleiffischer @kidder